.gitignore

```diff
@@ -17,3 +17,6 @@ config.yaml
 /sources/
 /sinks/
 /src/version.rs
+
+### IDE ###
```

Comment:
As a best practises, this should be set using your global git ignore file https://gist.github.com/subfuzion/db7f57fff2fb6998a16c, it should be located at `~/.config/git/ignore`